release/mkrelease.js

```diff
@@ -0,0 +1,306 @@
+const fs = require('fs');
+const request = require('request');
```

Comment:
You don't need to change this, but in general I think we should probably prefer libraries/patterns that allow for async/await. Notably in this case, we could've used our own stuff - https://github.com/microsoft/typed-rest-client - which is also good for dogfooding. Async/await lets us avoid the callbacks, which can get messy pretty fast (and kinda do in this PR IMO, its a little hard to follow)

Reply:
If you do want to do this, but with minimal changes, it looks like there are some wrappers around `request` that can get the job done - https://www.npmjs.com/package/request#promises--asyncawait

I'm ok as is though.

Reply:
Danny's comment makes sense. Let's schedule moving to the typed rest client for practice.

Reply:
The conversion was pretty straight forward and was happy to get rid of the nested callbacks.